QUICKSTART.md

```diff
@@ -0,0 +1,165 @@
+# NgRX Auto-Entity
+
+Automatic entities for [@ngrx](https://github.com/ngrx/platform)! With this library, you can take the boiler off your plate, and
```

Comment:
I do like the levity here, but "boilerplate" has become a pretty touchy subject around NgRx. Mike and Brandon have been drilling it into people's heads that it's not actually boilerplate (and in my opinion going just short of saying "you're dumb if you think this is boilerplate"). Since we're a sponsor of the NgRx project, we probably want to avoid directly contradicting them. 

Maybe we need to find a different way to word boilerplate, or use quotes when referring to "boilerplate". What do you guys think?

Reply:
maybe something along these lines?

With this library you can abstract the explicitness of common NgRx functionality similar to the @NgRx EntityAdapter while avoiding a black box of functionality and retaining complete control.

Reply:
I know they are touchy about it, however I think they are trying to redirect the narrative as well, and are pushing one particular (and not necessarily the best) philosophy about code development. I understand their "be verbose, be clear" argument, but in the real world there is more to it than that. The simple fact of the matter is, NGRX requires massive repetitive code implementation. IMO, unnecessarily so (which I think we can demonstrate quite clearly with Auto Entity). The schematics don't really help, as all they do is create a mostly empty file with the most bare-bones of implementation, requiring the developer to still spend a significant amount of time implementing code that really isn't critically necessary for clarity in the codebase. (Again, I think Auto Entity can prove my argument here.) 

There IS a high boilerplate requirement with NGRX, most developers would use the term "boilerplate" to describe it, and I find it a little sad that they are trying to redefine terms and argue one particular philosophy than addressing the underlying problem. 

I am open to alternative wording here. @schuchard I was hoping for something short and catchy, rather than just descriptive. @schuchard @anthonymjones A bit of an elevator pitch sort of deal and yes, with some levity. ;)